src/storage/src/hummock/compactor/iterator.rs

```diff
@@ -86,12 +86,12 @@ impl SstableStreamIterator {
     pub fn new(
         block_metas: Vec<BlockMeta>,
         sstable_info: SstableInfo,
-        existing_table_ids: HashSet<StateTableId>,
         stats: &StoreLocalStatistic,
         task_progress: Arc<TaskProgress>,
         sstable_store: SstableStoreRef,
         max_io_retry_times: usize,
     ) -> Self {
+        let existing_table_ids = HashSet::from_iter(sstable_info.table_ids.iter().cloned());
```

Comment:
nits: let's name it sst_table_ids to make things more explicit. People familiar with the previous codes can easily be confused and think existing_table_ids is the table ids of the compaction group.